src/pkg/cluster/pvc.go

```diff
@@ -10,6 +10,9 @@ import (
 )
 
 // UpdateGiteaPVC updates the existing Gitea persistent volume claim and tells Gitea whether to create or not.
+// REVIEW(mkcp): It looks like we've got some error+signal coming back from these functions where we return a both a
```

Comment:
Review request (opt @phillebaba)

Reply:
Your analysis is correct. The problem is that the caller of this function expects to have a string to print to STDOUT which is then  consumed by something else that is calling the CLI. Until we can fix the underlying implementation keeping the TODO is all we can do.

Reply:
Alright i'm going to swap this over to a TODO that summarizes